style.css

```diff
@@ -412,3 +412,158 @@ textarea {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media (min-width: 768px) {
+  .main-div {
+    border-radius: 0 0 0 150px;
+  }
+
+  .header {
+    padding: 25px 12%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: #6070ff;
+  }
```

Comment:
- Kindly, make your navigation responsive as given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)👌👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206303472-d1007669-44e5-4028-941d-bf035ae86367.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206303483-55a651b7-ec3b-49ea-9b71-1bdaa52b7876.png" width="200">|

😎️
Kindly, make sure you stick to the Figma design.